backend/api/Controllers/MissionSchedulingController.cs

```diff
@@ -285,7 +285,8 @@ ISourceService sourceService
             {
                 missionDeck ??= missionArea.Deck;
 
-                if (missionDeck != missionArea.Deck) { return BadRequest("The mission spans multiple decks"); }
+                if (missionDeck != missionArea.Deck)
+                    logger.LogWarning($"Mission {echoMission.Name} has tags in both deck {missionDeck.Name} and {missionArea.Deck.Name}.");
```

Comment:
Will this ensure that we select the deck with the most tags?

Reply:
No, but I can try implement that.

Reply:
Is it maybe not best to just use the area belonging to the area in the request body? I think it might get very confusing otherwise in the frontend, and it sounds like what you'd expect when using the endpoint. It is anyways what we do now and have done previously

Reply:
Might be enough to just select area with the most tags. Will in most cases also be the deck with the most tags

Reply:
I have updated the issue to explain that I am not addressing this issue in this PR, so I will also resolve this conversation.